cpp/custrings/tests/test_convert.cu

```diff
@@ -127,15 +129,17 @@ TEST_F(TestConvert, ToFloat)
         double expected[] = { 1234.0, 0,
             -876.0, 543.2, -0.12, 0.25, -0.002,
             0, nanval, 0, 123.0, 456.0, -178000.0,
-            -122.3364478212345, infval };
+            -122.33644782123469, infval };

```

Comment:
Why not use the same `-122.33644782123456789` as above, and let the compiler truncate to float/double?

Reply:
For some reason, this did not work with with `EXPECT_DOUBLE_EQ`.
I've changed it to `EXPECT_NEAR` now so I think that may be ok.